lib/client/src/components/profile/StaticProfile.js

```diff
@@ -82,52 +88,237 @@ const styles = {
   }
 };
 
-const StaticProfile = props => {
-  const {
-    classes,
-    profile: { username, createdAt, imageUrl, bio, website }
-  } = props;
+class StaticProfile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {}
+    };
+  }
+  componentDidMount() {
+    console.log("my state in static profile :: ", this.state);
+    console.log("my props in static profile :: ", this.props);
+    axios
+      .get("/user")
+      .then(res => {
+        console.log(res.data);
 
-  return (
-    <Paper className={classes.paper}>
-      <div className={classes.profile}>
-        <div className="image-wrapper">
-          <img src={imageUrl} alt="profile" className="profile-image" />
-        </div>
-        <hr />
-        <div className="profile-details">
-          <MuiLink
-            component={Link}
-            to={`/users/${username}`}
-            color="primary"
-            variant="h5"
-          >
-            @{username}
-          </MuiLink>
-          <hr />
-          {bio && <Typography variant="body2">{bio}</Typography>}
+        this.setState({
+          user: res.data
+        });
+        console.log("i have just mounted and this is my state : ", this.state);
+        window.location.reload(false);
```

Comment:
Here is where I gave up and reloaded the entire page after changing the state to what was updated in the database. Should it even go in a componentDidMount ?